packages/wdio-config/src/lib/ConfigParser.js

```diff
@@ -256,6 +265,26 @@ export default class ConfigParser {
         return this._capabilities
     }
 
+    static removeLineNumbers(pattern) {
+        if (pattern.includes(':')) {
+            pattern = pattern.split(':')[0]
+        }
+        return pattern
+    }
+
+    static isCucumberFeatureWithLineNumber(patterns) {
+        let isCucumberFeatureWithLineNumber = false
+        if (typeof patterns === 'string') {
+            isCucumberFeatureWithLineNumber = patterns.include(':')
```

Comment:
You can return immediatelly

```suggestion
            return patterns.include(':')
```

Reply:
This creates an `eslint` error when trying to push or commit.

Reply:
NM, other comments address that!

Reply:
Done! :-)